### PR TITLE
chore: redefine measurement manage api

### PR DESCRIPTION
**Pull request by xuthus5**

### CODE REVIEW

File: src/zh/guide/develop/client_design.md [hunk 1]
```diff
@@ -68,8 +68,9 @@ classDiagram
         + RetentionPolicy[] ShowRetentionPolicies(String database)
         + void DropRetentionPolicy(String database, String retentionPolicy)
         + RetentionPolicy[] ShowRetentionPolicies(String database)
-        + String[] ShowMeasurements(String database, String retentionPolicy)
+        + void CreateMeasurement(CreateMeasurementBuilder builder)
         + void DropMeasurement(String database, String retentionPolicy, String measurement)
+        + String[] ShowMeasurements(ShowMeasurementBuilder builder)
```
Comment (shoothzj) [inline src/zh/guide/develop/client_design.md:73]: Please provide the builder define

File: src/guide/develop/client_design.md [hunk 2]
```diff
@@ -67,8 +67,9 @@ classDiagram
         + RetentionPolicy[] ShowRetentionPolicies(String database)
         + void DropRetentionPolicy(String database, String retentionPolicy)
         + RetentionPolicy[] ShowRetentionPolicies(String database)
-        + String[] ShowMeasurements(String database, String retentionPolicy)
+        + void CreateMeasurement(CreateMeasurementBuilder builder)
         + void DropMeasurement(String database, String retentionPolicy, String measurement)
+        + String[] ShowMeasurements(ShowMeasurementBuilder builder)
```
Comment (shoothzj) [inline src/guide/develop/client_design.md:72]: Please update the method order to follow the sequence: Create, Show, Delete.

File: src/guide/develop/client_design.md [hunk 3]
```diff
@@ -67,8 +67,9 @@ classDiagram
         + RetentionPolicy[] ShowRetentionPolicies(String database)
         + void DropRetentionPolicy(String database, String retentionPolicy)
         + RetentionPolicy[] ShowRetentionPolicies(String database)
-        + String[] ShowMeasurements(String database, String retentionPolicy)
+        + void CreateMeasurement(CreateMeasurementBuilder builder)
```
Comment (shoothzj) [inline src/guide/develop/client_design.md:70]: There are issues with lines 67 and 69, as they are duplicated. Couly you please resolve this?

**Review by weiping-code — APPROVED** [review]

File: src/guide/develop/client_design.md [hunk 4]
```diff
@@ -76,6 +76,35 @@ classDiagram
         + String ShardGroupDuration
         + String IndexDuration
     }
+    class CreateMeasurementBuilder {
+        + CreateMeasurementBuilder TagList(String[] tags)
```
Comment (shoothzj) [inline src/guide/develop/client_design.md:80]: I believel Tags is better than TagList

File: src/guide/develop/client_design.md [hunk 5]
```diff
@@ -76,6 +76,35 @@ classDiagram
...
+        + CreateMeasurementBuilder FieldMap(map[String]fieldType fields)
+        + CreateMeasurementBuilder ShardKeys(String[] shardKeys)
+        + CreateMeasurementBuilder ShardType(shardType shardType)
```
Comment (shoothzj) [inline src/guide/develop/client_design.md:83]: put `ShardType` behind `ShardKeys` will be better?

File: src/guide/develop/client_design.md [hunk 6]
```diff
@@ -76,6 +76,35 @@ classDiagram
...
+        + CreateMeasurementBuilder EngineType(engineType engineType)
+        + CreateMeasurementBuilder PrimaryKey(String[] primaryKeys)
+        + CreateMeasurementBuilder SortKeys(String[] sortKeys)
+        + String build()
+    }
+    class ShowMeasurementBuilder {
+        + ShowMeasurementBuilder Filter(ComparisonOperator operator, String regex)
+        + String build()
+    }
+    class fieldType {
```
Comment (shoothzj) [inline src/guide/develop/client_design.md:95]: why this is lower case?

**Review by shoothzj — APPROVED** [review]

**Pull request merged**